perfmetrics/scripts/testing_on_gke/examples/fio/run_tests.py

```diff
@@ -15,39 +15,70 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""This program takes in a json fio test-config file and generates and deploys helm charts."""
+
+import argparse
 import subprocess
+import fio_workload
+
 
 def run_command(command: str):
-    result = subprocess.run(command.split(" "), capture_output=True, text=True)
-    print(result.stdout)
-    print(result.stderr)
-
-bucketName_fileSize_blockSize = [
```

Comment:
Replacement for this hardcoded set of tests is in https://github.com/GoogleCloudPlatform/gcsfuse/pull/2264/files .